frontend/src/scenes/events/eventsTableLogic.js

```diff
@@ -181,12 +181,19 @@ export const eventsTableLogic = kea({
 
     actionToUrl: ({ values }) => ({
         setProperties: () => {
-            return [router.values.location.pathname, values.propertiesForUrl, window.location.hash]
+            return [
+                router.values.location.pathname,
+                {
+                    ...router.values.searchParams,
+                    properties: values.properties,
+                },
+                window.location.hash,
+            ]
```

Comment:
If this makes `propertiesForUrl` unnecessary, we should remove it.

Reply:
I don't see anywhere we use the `eventsTableLogic.values.propertiesForUrl` anyways so I think this can be removed.